bundles/org.openhab.binding.visualcrossing/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,1774 @@
+		<state readOnly="true" min="0" max="100"/>
+	</channel-type>
+	<channel-type id="precip-type-channel">
+		<item-type>String</item-type>
+		<label>Precip Type</label>
+		<description>
+			An comma separated array indicating the type(s) of precipitation expected or that occurred. Possible
+			values include `rain`, `snow`, `freezingrain` and `ice`.
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="precip-cover-channel">
+		<item-type>Number</item-type>
+		<label>Precip Cover</label>
+		<description>The proportion of hours where there was non-zero precipitation</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="snow-channel">
+		<item-type>Number:Length</item-type>
+		<label>Snow</label>
+		<description>The amount of snow that fell or is predicted to fall</description>
+		<category>Snow</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="snow-depth-channel">
+		<item-type>Number:Length</item-type>
+		<label>Snow</label>
+		<category>Snow</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="visibility-channel">
+		<item-type>Number:Length</item-type>
+		<label>Visibility</label>
+		<description>Distance at which distant objects are visible</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="cloud-cover-channel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Cloud Cover</label>
+		<description>How much of the sky is covered in cloud ranging from 0–100%</description>
+		<state readOnly="true" min="0" max="100"/>
+	</channel-type>
+	<channel-type id="solar-radiation-channel">
+		<item-type>Number:Intensity</item-type>
+		<label>Solar Radiation</label>
+		<description>The solar radiation power at the instantaneous moment of the observation (or forecast prediction).
+			See the
+			full solar radiation data documentation
+			(https://www.visualcrossing.com/resources/documentation/weather-data/how-to-obtain-solar-radiation-data/)
+			and Wind and
+			Solar Energy
+			(https://www.visualcrossing.com/resources/documentation/weather-api/energy-elements-in-the-timeline-weather-api/)
+			pages.
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="solar-energy-channel">
+		<!-- TODO Is there item-type for MJ /m^2 ? -->
+		<item-type>Number</item-type>
+		<label>Solar Energy</label>
+		<description>(MJ /m^2) Indicates the total energy from the sun that builds up over an hour or day. See the full
+			solar
+			radiation data documentation
+			(https://www.visualcrossing.com/resources/documentation/weather-data/how-to-obtain-solar-radiation-data/)
+			and Wind and
+			Solar Energy
+			(https://www.visualcrossing.com/resources/documentation/weather-api/energy-elements-in-the-timeline-weather-api/)
+			pages.
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="uv-index-channel">
+		<!-- TODO Is there item-type for UV? -->
+		<item-type>Number</item-type>
```

Comment:
@lsiepel Is there item-type for UV?

Reply:
Since openHAB 4.2.0 there is a system channel for that: 
For reference
https://github.com/openhab/openhab-core/pull/3878

Seems the docs have not been updated yet.